README.md

```diff
@@ -1 +1,198 @@
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Yoy can use this repository ising the following steps:
+           
+           1. Clone a repository from www.github.com You can use the follonig link directly to the repository:
+           https://github.com/jovasal/Capstone.git
+           
+           2.Clone this repository to your desired folder:
+           
+           Example commands:
+           ```sh
+           cd my-folder
+           git clone https://github.com/jovasal/Capstone.git
+           ```
+           
+           If you prefer to download without commands, tin the link, 
+           you can press <> Code button, choose Local and then Download.zip. 
+           Remember to extrat in your faverite part of your computer.
+           
+           Finaly, you can use as a reference for your future codes.
+
+### Install
+
+Is not necesary to install, but you need any sofware that you can use HTML and CSS
+
+### Usage
+
+ 1. Clone a repository from www.github.com You can use the follonig link directly to the repository:
+           https://github.com/jovasal/Capstone.git
+           
+           2.Clone this repository to your desired folder:
+           
+           Example commands:
+           ```sh
+           cd my-folder
+           git clone https://github.com/jovasal/Capstone.git
+           ```
+
+### Run tests
+
+Test is in the following repository: https://github.com/jovasal/Capstone.git
+
+### Deployment
+
+
+All deployment is in: https://github.com/jovasal/Capstone.git
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors">John Alonso Vasquez Salcedo</a>
+
+John Alonso Vasquez Salcedo
+
+> Thank you so mucho to Microverse and FIGMA
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/jovasal/)
+- Twitter: [@twitterhandle](https://twitter.com/jovasal)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/john-alonso-vasquez-salcedo-95749632/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[JAVASCRIPT]**
+- [ ] **[CSS]**
+- [ ] **[HTML]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/jovasal/Capstone.git/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project kindly show some love, give it a 🌟 **STAR** 🌟
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Thank you so much to Miroverse course and FIGMA
```

Comment:
nice work so far 👍🏿  i n this section it is recommended to give acknowledgment to the author of the design  [Cindy Shin](https://www.behance.net/adagio07)

Reply:
Great!!

it was changed for:

Original design idea by Cindy Shin: https://www.behance.net/adagio07